docs/introduction/integration_test.rst

```diff
@@ -336,6 +336,9 @@ Flow
 6. If the tests fail, you will see it in the the output. For more
    details, you can take a look at the log file (logs to STDOUT as default).
 
+.. note:: User can run several test using several times command -c. If user precises a folder path, different log files
+will be saved in the folder, if user precise a file path, then it will append all the logs in this file.
```

Comment:
If a folder path is specified, a log for each yaml file will be stored. If otherwise a filename is provided, all log information will be in one logfile. 

précises looks more like a french thing.  (:

Reply:
Oupsi, I'll change it :)